### PR TITLE
Bib 767 update status naming convention

### PR DESCRIPTION
**Pull request by kaseywright**

### CODE REVIEW

File: src/lib/components/ProjectProgressBar.svelte [hunk 1]
```diff
@@ -22,20 +23,20 @@
...
-                <div>Not Started</div>
+                <div>Awaiting AI Draft</div>
             </div>
             <div class="flex flex-row space-x-1">
                 <div class="h-4 w-5 border border-black bg-primary bg-opacity-25" />
-                <div>In Progress</div>
+                <div>In Editor Review</div>
             </div>
             <div class="flex flex-row space-x-1">
                 <div class="h-4 w-5 border border-black bg-primary bg-opacity-60" />
```
Comment (MrGrinst) [inline src/lib/components/ProjectProgressBar.svelte:42]: Looks like `In Manager Review` needs updated below

File: src/routes/(dashboard)/ManagerDashboard.svelte [hunk 2]
```diff
@@ -367,13 +367,13 @@
             {#if $searchParams.tab === Tab.myWork}
                 <Tooltip
                     position={{ left: '10rem' }}
-                    text={nonManagerReviewSelected ? 'Manager Review status only' : null}
+                    text={nonCompanyReviewSelected ? 'Manager Review status only' : null}
```
Comment (MrGrinst) [inline src/routes/(dashboard)/ManagerDashboard.svelte:370]: `Manager Review` needs updated here

File: src/routes/resources/[resourceContentId]/+page.svelte [hunk 3]
```diff
@@ -141,22 +141,22 @@
             resourceContent.status === ResourceContentStatusEnum.AquiferizePublisherReview;
 
         isInTranslationWorkflow =
-            resourceContent.status === ResourceContentStatusEnum.TranslationNotStarted ||
+            resourceContent.status === ResourceContentStatusEnum.TranslationAwaitingAiDraft ||
```
Comment (MrGrinst) [inline src/routes/resources/[resourceContentId]/+page.svelte:144]: I think we want to add `TranslationAiDraftComplete` here too

File: src/routes/projects/+page.svelte
Comment (MrGrinst) [inline src/routes/projects/+page.svelte:1]: Looks like there are still some `...ManagerReview` mentions in this file that need updating, unless we were planning on doing that separately?

Comment (kaseywright) [reply src/routes/projects/+page.svelte:1]: Many of these mentions were for a pulling back from Manager (Company) Review. I updated the names but it sounds like something that would violate the requirement that the status should never go back from `*Company Review`.

Comment (MrGrinst) [reply src/routes/projects/+page.svelte:1]: I think in this case it's going from `Pending Review` to `Company Review`, which is still functionality we want to support. Also responded to your comment in `aquifer-server` saying the same.

**Review by jwinston-bn — APPROVED** [review]

**Pull request merged**